main.js

```diff
@@ -263,27 +264,47 @@ class ModuleInstance extends InstanceBase {
 					true
 				)
 			} else if ((matches = message.address.match(cmd))) {
-				let user_id = matches[1]
-				if (user_id === this.config.user_id || this.config.user_id === '-1') {
+				let userid = matches[1]
+				if (userid == this.config.userid || this.config.userid == '-1') {
```

Comment:
```suggestion
				if (userid == this.config.user_id || this.config.user_id == '-1') {
```
In 2.4 the config field was `user_id`, so that name should be kept to avoid breaking existing installations.
The config field will need updating to match, I can't do that as a suggestion like this